tools/decompile.py

```diff
@@ -52,23 +52,19 @@ def __init__(self, func_root: Path, function_path: Path, args: argparse.Namespac
     def overlay(self) -> str:
         """Attempts to infer the overlay from the asm file path or returns None if one is not found.
         The '_psp' extension will be included if the version is psp"""
-        nonmatchings_dir = (
-            "f_nonmat"
-            if args.version == "saturn"
-            else "psp" if "psp" in args.version else "nonmatchings"
-        )
+        nonmatchings_dir = "f_nonmat" if args.version == "saturn" else "nonmatchings"
```

Comment:
Previously extracted overlays still use psp as the nonmatchings directory, this will break the tool for them.

Reply:
> All the PSP overlays produced by make-config.py use nonmatchings in their path.

From my PR description.

Reply:
I'm confused, are you saying that we don't need to care about being able to use `decompile.py` on overlays that use `psp` as `nonmatchings_dir`?

Reply:
I am not sure why some of the existing overlays use a different nonmatchings directory. These overlays need to be adjusted accordingly. We have tools that relies on that being part of the path.

Reply:
This I definitely agree on, I actually raised this question in discord a week ago: https://discord.com/channels/1079389589950705684/1087866009983139870/1358808728257953967